sdk/tests/client/node_api/mod.rs

```diff
@@ -96,9 +96,9 @@ pub async fn setup_transaction_block(client: &Client) -> (BlockId, TransactionId
         _ => unreachable!(),
     };
 
-    let _ = client.retry_until_included(&block.id(), None, None).await.unwrap();
+    let retried = client.retry_until_included(&block.id(), None, None).await.unwrap();
```

Comment:
I know it's just a test ... but the binding name is a bit lame .... `retried_block`?

Reply:
Can be more than one, `retried_blocks`? Then please do a code suggestion and just merge